lib/youtex/types.ex

```diff
@@ -1,14 +1,21 @@
 defmodule Youtex.Types do
   @moduledoc false
 
-  alias Youtex
+  alias Youtex.Transcript.Sentence
+  alias Youtex.Transcript
+  alias Youtex.Video
 
   defmacro __using__(_opts) do
     quote do
+      @type video :: Video.t
       @type video_id :: String.t
       @type language :: String.t
 
-      @type success :: {:ok, [Youtex.t]}
+      @type transcript_list :: list(Transcript.t)
+      @type sentence_list :: list(Sentence.t)
```

Comment:
👍🏻

Reply:
What's the difference between `list()` and `[]`? We can pick `list(x)` as our coding-style for this project and use only it. :)

Reply:
> What's the difference between `list()` and `[]`? We can pick `list(x)` as our coding-style for this project and use only it. :)

I guess there are some small differences regarding proper/improper list or regarding emptyness, I guess. That's something I'm not sure.

Good, we'll try to keep the same approach.